specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "api-version": "2020-06-02",
```

Comment:
The `"api-version"` should be `"2021-03-01"`

Reply:
are you sure, see https://github.com/Azure/azure-rest-api-specs/blob/8c5be206f106d19aeb87ca447e730b936eb308b1/specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/examples/CreateBot.json#L5

Reply:
I see, let's wait for the Azure REST API team to comment on it. I looked at specs from other teams and the versions matched, but this also makes sense.

The [`GetHostSettings` example](https://github.com/Azure/azure-rest-api-specs/pull/12995/files#diff-dacf7c1af9183cf7a466f0c942f1850634608e9605c2c9db2965d2b2a12d5cba) has this new version as expected.

Reply:
based on https://stackoverflow.microsoft.com/questions/245190, I have updated the api-version in the new examples folder